src/components/App/App.js

```diff
@@ -77,33 +75,31 @@ class App extends Component {
   }
   
   render() {
-    console.log('SELECTED TREE AT THE ACTUAL END', this.state.selectedTree)
-    // const page = this.state.selectedZip && this.state.selectedTree ? '/details' : '/result' 
-    // page = selectedTree ? '/details' : '/result'
-    // console.log(page, "PAGE71")
     return (
       <>
-        <Header clearZipTrees={this.clearZipTrees}/>
         <Switch>
-          {/* <Route exact path='/' render={() => <Home changeZipCode={this.changeZipCode}/>}/> */}
-          <Route exact path="/">
-	          {this.state.selectedZip ? <Redirect to="/result" /> : <Home changeZipCode={this.changeZipCode}/>}
+          <Route 
+            exact path='/'>
+	          {this.state.selectedZip ? <Redirect to='/result' /> : <Home changeZipCode={this.changeZipCode}/>}
           </Route>;
-          <Route exact path="/result">
-	          {this.state.selectedTree ? <Redirect to="/trees/:id" /> : <SearchResult filteredTrees={this.state.filteredTrees} changeSelectedTree={this.changeSelectedTree} clearZipTrees={this.clearZipTrees}/>}
+          <Route 
+            exact path='/result'>
+	          {this.state.selectedTree ? <Redirect to='/trees/:id' /> : <SearchResult city={this.state.city} filteredTrees={this.state.filteredTrees} changeSelectedTree={this.changeSelectedTree} clearZipTrees={this.clearZipTrees}/>}
```

Comment:
Hooray we can see the city associated with the zip code! Makes for a much better UI after you submit the zip code to double check submission was done right! 🥳